From 1d3ef72dee804cb3a39c5b536accf5a4480adc20 Mon Sep 17 00:00:00 2001
In-Reply-To: <1494009820-2090-3-git-send-email-bnmvco@gmail.com>
References: <1494009820-2090-3-git-send-email-bnmvco@gmail.com>
From: Daniel Ferreira <bnmvco@gmail.com>
Date: Fri, 5 May 2017 15:43:40 -0300
Subject: [TOSQUASH PATCH 3/4] add--interactive: use add-interactive--helper
 for status_cmd

Call the newly introduced git-add-interactive--helper builtin on
status_cmd() instead of relying on git-add--interactive's Perl
functions to build print the numstat.

Signed-off-by: Daniel Ferreira <bnmvco@gmail.com>
---
 git-add--interactive.perl | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/git-add--interactive.perl b/git-add--interactive.perl
index 709a5f6ce6..86174624c6 100755
--- a/git-add--interactive.perl
+++ b/git-add--interactive.perl
@@ -632,9 +632,7 @@ EOF
 }
 
 sub status_cmd {
-	list_and_choose({ LIST_ONLY => 1, HEADER => $status_head },
-			list_modified());
-	print "\n";
+	system(qw(git add-interactive--helper --status));
 }
 
 sub say_n_paths {
-- 
2.11.0

